cmd/version.go

```diff
@@ -12,15 +12,15 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package main
+package cmd
 
 import (
 	"fmt"
 	"runtime"
 )
 
-// Set with `-ldflags -X main.gcsfuseVersion=1.2.3` by tools/build_gcsfuse. If
-// not defined, we use "unknown" in getVersion.
+// Set with `-ldflags -X github.com/googlecloudplatform/gcsfuse/v2/cmd.gcsfuseVersion=1.2.3`
```

Comment:
I would rather suggest to keep this file unchanged and at the same path as it was (outside of cmd directory). Would it be okay if we keep this file as it was ?

Reply:
Not really - as discussed offline, a package cannot refer to the main package since it introduces cyclic dependency.
Another approach could be to pass the version from the main command to cmd but it seems to be too much of an effort.
So, we will keep it as is.